app/src/main/res/values-sv-rSE/strings.xml

```diff
@@ -636,11 +636,24 @@
     <!-- This string is displayed as the title of the Bookmarks List, which contains the user's bookmarked websites. -->
     <string name="bookmarks_title">Bokmärken</string>
 
+    <!-- This string is displayed as the title of the Bookmarks List when the bookmarks list is empty. -->
+    <string name="bookmarks_empty">Din bokmärkeslista är tom</string>
+
+    <!-- This string is displayed under the bookmarks title in the bookmarks list. -->
+    <string name="bookmarks_description">Klicka på @-ikonen när du vill lägga till den i dina bokmärken.</string>
```

Comment:
Is the "-" specific to some languages? I see that most of them don't use it but some do.